tensorflow/lite/micro/examples/hello_world/Makefile.inc

```diff
@@ -37,6 +37,8 @@ $(HELLO_WORLD_TEST_SRCS),$(HELLO_WORLD_TEST_HDRS)))
 $(eval $(call microlite_test,output_handler_test,\
 $(OUTPUT_HANDLER_TEST_SRCS),$(OUTPUT_HANDLER_TEST_HDRS)))
 
+$(eval $(call generate_project_third_party_parsing))
```

Comment:
Are you able to include this generate_project_third_party_parsing function as part of the call microlite_test,hello_world and other example rules? As it stands this line would need to be added to any new examples, and it's not clear to developers why it's needed or what it does, so it would be preferable if we can include it under the hood instead.

Reply:
Yeah, I was thinking the same thing but wasn't sure what the best way to go about it was to prevent it from being called multiple times or at the wrong time. I think I have a good method now, which is to use the sort function of make which removes duplicates, which was the main issue. I Posted an update.